docs/index.md

```diff
@@ -71,6 +71,34 @@ containerd will come with a default implementation for the various components.
 These defaults will be chosen by the maintainers of the project and should not change unless better tech for that component comes out.
 Additional implementations will not be accepted into the core repository and should be developed in a separate repository not maintained by the containerd maintainers.
 
+### Build it
+You can build `containerd` via Docker container. You can build an image from
+this Dockerfile:
+
```

Comment:
Wouldn't it be better to `RUN make` within Dockerfile, and exec `docker run .. tar cf ...`.
It works even with remote daemon where bind-mount does not work as expected.